testdata/no-leader/inputs.json

```diff
@@ -8,7 +8,7 @@
       "MinQuorum": 3,
       "ServerStabilizationTime": 10000000000
    },
-   "State": {
+   "CurrentState": {
```

Comment:
All of these testdata changes are to fixup the name of the field. Before this change this field would be silently discarded when the test inputs were being loaded.